lib/Cache.js

```diff
@@ -30,20 +26,12 @@ class Cache {
 		};
 	}
 
-	getModule(identifier, callback) {
-		this.hooks.getModule.callAsync(identifier, callback);
+	get(identifier, etag, callback) {
```

Comment:
What about returning a `Promise` instead? The API would be friendlier and would allow `async/await` in the future.

Reply:
I thought about that, but I went with callbacks for these reasons:

* It's consistent with other webpack code which uses callbacks
* Callbacks are faster compared to promises and this code is called for every module
* You still can use promises in the plugin hook, because tapable allows both (tapPromise)